src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -95,10 +106,15 @@ case class OreAggregation(
                            originalRecord: ExactlyOne[String], //map v4 specifies this as a ref, but that's LDP maybe?
                            hasView: ZeroToMany[EdmWebResource] = Seq(),
                            intermediateProvider: ZeroToOne[EdmAgent] = None,
-                           `object`: ZeroToOne[EdmWebResource] = None, //full item page
-                           preview: ZeroToOne[EdmWebResource] = None, ///thumbnail
+                           `object`: ZeroToOne[EdmWebResource] = None, // full size image (should *not* be used)
```

Comment:
I think the take away is we do want to use this if available in the raw data, but generally thus far it is not.